buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkDimensions.kt

```diff
@@ -0,0 +1,113 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.skyscanner.backpack.tokens
+
+import com.google.common.base.CaseFormat
+import com.squareup.kotlinpoet.*
+import javax.script.ScriptEngineManager
+
+interface BpkDimensions : Map<String, Int>
+
+object BpkDimension {
+
+  sealed class Category : BpkParser<BpkDimensions> {
+
+    object Spacing : Category() {
+      override fun invoke(source: Map<String, Any>): BpkDimensions =
+        parseDimensions(source, "spacings", "SPACING_") {
+          it.key.toLowerCase() in setOf("base", "lg", "md", "none", "sm", "xl", "xxl")
```

Comment:
I wonder why that is and if that's a good thing (similar to the comment above) - surely the tokens should be the source of truth, rather than us having to filter what we know about?